modules/build.gradle

```diff
@@ -22,13 +22,13 @@ ext {
 	projectTemplateTokens = [
 		"biz.aQute.bnd.version": "3.5.0",
 		"com.liferay.ant.bnd.version": "2.0.52",
```

Comment:
Should these be bumped?

Reply:
Peter wants me to remove this actually. He said Shuyang removed it from portal

Reply:
Remove ant.bnd?  All of those plugins are still needed right?